pawn/Driver/Driver.pwn

```diff
@@ -224,12 +259,90 @@ public OnPlayerKeyStateChange(playerid, newkeys, oldkeys) {
             if (VehicleModel(modelId)->isNitroInjectionAvailable())
                 AddVehicleComponent(vehicleId, 1010);
         }
+
+        new bool: rightBlinker = g_blinkerObjects[playerid][0] != 0;
+        new bool: leftBlinker = g_blinkerObjects[playerid][2] != 0;
+        // Vehicle keys (q): blinker left
+        if (PRESSED(VEHICLE_KEYS_BINDING_BLINKER_RIGHT) && 
+            (vehicleKeys & VEHICLE_KEYS_BLINKER_RIGHT)) {
+
+            rightBlinker = !rightBlinker;
+        }
+
+        // Vehicle keys (e): blinker right
+        if (PRESSED(VEHICLE_KEYS_BINDING_BLINKER_LEFT) && 
+            (vehicleKeys & VEHICLE_KEYS_BLINKER_LEFT)) {
+
+            leftBlinker = !leftBlinker;
+        }
+
+        SetBlinker(playerid, vehicleId, leftBlinker, rightBlinker);
     }
 
     LegacyPlayerKeyStateChange(playerid, newkeys, oldkeys);
     return 1;
 }
 
+// Enable the |left| and or |right| blinkers for |playerid| in |vehicleId|
+SetBlinker(playerid, vehicleId, bool:left, bool:right) {
+    new const blinkerModel = 19294;
+    new const modelId = GetVehicleModel(vehicleId);
+
+    if(IsModelBoat(modelId) || IsModelPlane(modelId)) {
+        return;
+    }
+
+    new Float:sizeX, Float:sizeY, Float:sizeZ;
+    GetVehicleModelInfo(modelId, VEHICLE_MODEL_INFO_SIZE, sizeX, sizeY, sizeZ);
+
+    if (right) {
+        if (g_blinkerObjects[playerid][0] == 0) {
+            g_blinkerObjects[playerid][0] = CreateObject(blinkerModel, 0, 0, 0, 0, 0, 0, 0);
```

Comment:
nit: please use `CreateDynamicObject()`, and just stash `Dynamic` in the other function names related to the object. Then it uses the streamer, which gives us the much higher object limits, and it makes sure that players who are far away from this vehicle don't need to know about the indicator.